packages/expo-file-system/src/next/FileSystem.ts

```diff
@@ -60,6 +61,14 @@ export class File extends ExpoFileSystem.FileSystemFile {
   get name() {
     return Paths.basename(this.uri);
   }
+
+  readableStream() {
+    return new ReadableStream(new FileSystemReadableStreamSource(super.open()));
```

Comment:
Does Hermes support readable and writable streams? 😮

Reply:
Unfortunately it was a leak from `web-streams-polyfill`. Since expo-fetch and expo both depend on it, how about I add it as an explicit dependency instead of relying on global?